### ISSUE TITLE
Generate a code playground

### ISSUE THREAD

**Issue by majkl-zumberi** [ngryman/reading-time#47]
it would be nice to provide a code playground of this library, in order to test it with no effort & check  if it suits the needs of those who want to use Reading-time lib

**Comment by Josh-Cena** [comment]
Good idea! 👍 As a maintainer of [Docusaurus](https://docusaurus.io/) we are definitely in a good position to set up a website w/ playground 😁

**Comment by arpitBhalla** [comment]
@Josh-Cena May I work on this?

**Comment by Josh-Cena** [comment]
@arpitBhalla I'll be working on this. We will be creating an entire website from scratch, so it's a bit non-trivial. You can send polishing PRs afterwards if you want to contribute.

**Comment by ngryman** [comment]
How about providing a link to stackblitz in the README with various text examples?

Here's an example from another PR: https://stackblitz.com/edit/node-nyf9tn?file=text.js.

**Comment by arpitBhalla** [comment]
I am still open to contribute to playground website.

**Comment by ngryman** [comment]
@arpitBhalla That'd be awesome!
